### PR TITLE
Convert README.md to rst with pypandoc for PyPI readability

### PR DESCRIPTION
**Pull request by aprotopopov**
For now package [description](https://pypi.python.org/pypi/Lifetimes) on PyPI is unreadable. Added convert from .md to .rst with `pandoc` and `pypandoc` as stated in [SO answer](https://stackoverflow.com/a/26737672/4542359). 
Couldn't verify that on local machine may be you could do that?

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -6,8 +6,13 @@
              'lifetimes/version.py', 'exec'))
 
 
-def read(fname):
-    return open(os.path.join(os.path.dirname(__file__), fname)).read()
+readme_path = os.path.join(os.path.dirname(__file__), 'README.md')
+
+try:
+    import pypandoc
+    long_description = pypandoc.convert(readme_path, 'rst')
```
Comment (CamDavidsonPilon) [inline setup.py]: `convert_file` is the current api

**Comment by CamDavidsonPilon** [comment]
Checking this, I have a few additions I want to make - can you enable edits by the maintainer on this pr please?

File: setup.py (hunk 1 above)
Comment (aprotopopov) [inline setup.py]: Yes... changed it in docs/conf.py but forgot to change here...

**Comment by aprotopopov** [comment]
@CamDavidsonPilon edits from maintainers were enabled before. As I understand it's enabled by default or I did that long ago. Did you try to make some changes on that or other PR?

**Comment by CamDavidsonPilon** [comment]
Cool, my changes made it in :) 
I'm going to merge this, and we can rebase #114 against this. I predict that other PR will take longer, so no reason to complicate that one.

**Pull request merged**